1.19_linux/syscall/stubs_linux_amd64.go

```diff
@@ -0,0 +1,5 @@
+package syscall
```

Comment:
The filename should have a prefix `hitsumabushi_` so that we realize this file is a completely new file. `hitsumabushi_stubs_linux_amd64.go` is fine.